Extension/src/LanguageServer/client.ts

```diff
@@ -952,15 +977,31 @@ export class DefaultClient implements Client {
             const value: string | undefined = await vscode.window.showInformationMessage(localize("setCompiler.message", "You do not have IntelliSense configured. Unless you set your own configurations, IntelliSense may not be functional."), buttonMessage);
             secondPromptCounter++;
             if (value === buttonMessage) {
-                this.handleCompilerQuickPick(showSecondPrompt, sender);
+                this.handleIntelliSenseConfigurationQuickPick(showSecondPrompt, sender);
             }
         }
     }
 
-    public async handleCompilerQuickPick(showSecondPrompt: boolean, sender?: any): Promise<void> {
+    public async handleIntelliSenseConfigurationQuickPick(showSecondPrompt: boolean, sender?: any, compilersOnly?: boolean | undefined): Promise<void> {
```

Comment:
`| undefined` is not needed when the argument is `compilersOnly?`

Reply:
changed